src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java

```diff
@@ -0,0 +1,176 @@
+package com.dataguise.saas.dto;
+
+import com.google.gson.Gson;
+
+public class ColumnViewUIFormatterDTO {
+
+    String columnName;
+    String dataType;
+    Integer regexId;
+    String regexGroup;
+    String regexLabel;
+    Boolean isSensitive;
+    Boolean isProtected;
+    Boolean isUnprocessed;
+    Boolean isUnscanned;
+    Boolean isColumnHeaderMatched;
+    Boolean isCleaned;
+    Boolean isSafe;
+    Long hitCount;
+    Long mismatchCount;
+    Long nullCount;
+    Double expressionGroupConfidence;
+
+
+
+    String constraintType;
+
+    public String getColumnName() {
+        return columnName;
+    }
+
+    public void setColumnName(String columnName) {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.design.DesignForExtensionCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/ColumnViewUIFormatterDTO.java L32
    
    Class &#39;ColumnViewUIFormatterDTO&#39; looks like designed for extension (can be subclassed), but the method &#39;setColumnName&#39; does not have javadoc that explains how to do that safely. If class is not designed for extension consider making the class &#39;ColumnViewUIFormatterDTO&#39; final or making the method &#39;setColumnName&#39; static/final/abstract/empty, or adding allowed annotation for the method.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a468145800>*